utils/experiment/experiment.go

```diff
@@ -46,7 +46,27 @@ func IsTerminating(experiment *v1alpha1.Experiment) bool {
 			return true
 		}
 	}
-	return false
+	return CompletedAllRequiredAnalysisRuns(experiment, &experiment.Status)
+}
+
+// CompletedAllRequiredAnalysisRuns has at least one required for completition analysis run
+// and it completed successfully
```

Comment:
Can we rename this to: `RequiredAnalysisRunsSuccessful()` ? I didn't realize this would only only return when they were successful and not just completed.